README.md

```diff
@@ -1 +1,189 @@
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+4. Run
+   ```
+   npx hint .
+   ```
+5. Fix validation errors.
+
+### [Stylelint](https://stylelint.io/)
+
+A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
+
+1. Run
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+not sure how to use npm? Read this.
+
+2. Copy .stylelintrc.json to the root directory of your project.
+
+3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+
+If you think that change is necessary - open a Pull Request in this repository and let your code reviewer know about it. 4. Run npx stylelint "\*_/_.{css,scss}" on the root of your directory of your project.
+
+5. Fix linter errors.
+
+6. **IMPORTANT NOTE:** feel free to research auto-correct options for Stylelint if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: (https://github.com/siddghosh108)
+- Twitter: (https://twitter.com/siddharthaghos9)
+- LinkedIn: (https://www.linkedin.com/in/siddhartha-ghosh-65902718)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+      This project will not have any additional features.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly leave a comment below and share it with someone who enjoys coding! Coding is all about continuous learning and allowing yourself to be a beginner. Keep going!
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="Microverse Inc."></a>
+
+I'm grateful to microverse for giving me every opportunity to grow into my full potential
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]**
+  Do I have to use the vs code specifically?
+
+  - [Answer_1]
+    You can use any code editor of your chose.
+
+- **[Question_2]**
+  Where can I download node JS for installation?
+
+  - [Answer_2]
+  Node Js can be downloaded here- https://nodejs.org/en/download/
+  <!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly work more on your license mit ,your link is working but yuu need to remove some texts which is use to describe how you can edit to manage the license template please remove the unnecessary text so that it will look professional. Please watch this [tutorial](https://www.youtube.com/watch?v=8Dm7XXvtqXI) to know how to add one, remember to use the MIT license.

Please refer to the [MIcroverse README template](https://github.com/microverseinc/readme-template) to get a better understanding of what you need to do.

Reply:
I was not actually clear about what exactly you asked me to remove fromMIT.md but i removed the texts that told about editing or managing the licence